Utils/Settings.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Settings.swift
+//  VideoQuickStart
```

Comment:
This utils file doesn't belong to a particular project. Should we use VideoQuickStart or something else?